plugins/container/go-worker/pkg/container/cri.go

```diff
@@ -520,20 +518,21 @@ func (c *criEngine) Listen(ctx context.Context, wg *sync.WaitGroup) (<-chan even
 				}
 
 				var info event.Info
-				// verbose true to return container.Info
-				ctr, err := c.client.ContainerStatus(ctx, evt.ContainerId, true)
+				returnInfo := true
+				ctr, err := c.client.ContainerStatus(ctx, evt.ContainerId, returnInfo)
```

Comment:
@deepskyblue86 Am I getting it right that we're deducing it's a sandbox just because the call to `ContainerStatus` is failing?